linkerd/metadata.csv

```diff
@@ -38,3 +39,31 @@ linkerd.rt.client.request_latency_ms.quantile,gauge,,millisecond,,Stats of the l
 linkerd.rt.server.connections,gauge,,connection,,Number of active connections for the server,0,linkerd,server active connections
 linkerd.rt.server.connects_s,rate,,connection,second,Number of connection par second for the server,0,linkerd,server connects/s
 linkerd.rt.server.request_latency_ms.quantile,gauge,,millisecond,,Stats of the latency of requests in milliseconds for the server,-1,linkerd,server requests latency
+linkerd.request_total,count,,,,Total count of HTTP requests.,0,linkerd,
```

Comment:
```suggestion
linkerd.request_total,count,,request,,Total count of HTTP requests.,0,linkerd,
```

Would suggest using `request` since the unit exists (https://github.com/DataDog/dogweb/blob/prod/integration/system/units_catalog.csv)

Same feedback for other cases in this PR where requests is the unit.

Reply:
Done